code/script.js

```diff
@@ -244,100 +246,115 @@ const generateBoard = () => {
   })
 }
 
-// Randomly select a person from the characters array and set as the value of the variable called secret
+// Decides on the secret person...
 const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
 }
 
-// This function to start (and restart) the game
+// Starts the game
 const start = () => {
-  // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+  generateBoard()
+  setSecret()
+  winOrLose.style.display = 'none'
+
 }
 
-// setting the currentQuestion object when you select something in the dropdown
+// This is triggered by a change in the drop-down, and saves the selection as an object "currentQuestion"
 const selectQuestion = () => {
   const category = questions.options[questions.selectedIndex].parentNode.label
-  // This variable stores what option group (category) the question belongs to.
-  // We also need a variable that stores the actual value of the question we've selected.
-
+  const value = questions.options[questions.selectedIndex].value
   if (category === 'hair color') {
     currentQuestion = {
       attribute: 'hairColor',
-      // value: ,
-      // 👆 add the value from the input here
+      value: value,
```

Comment:
Here you could use the object property shorthand: "value," and "category," since the key and the value are the same.